app/middlewares/validators/users.js

```diff
@@ -39,3 +39,15 @@ exports.verifyAdmin = async (req, _, next) => {
   }
   return next();
 };
+
+exports.idVerify = async (req, _, next) => {
+  const { payload } = req.token;
+  const user = await User.findOne({ where: { email: payload } });
+
+  if (!user) {
+    return next(errors.notFoundError('UserId not found'));
+  }
+
+  req.userId = user.id; // eslint-disable-line
+  return next();
+};
```

Comment:
I don't think this is necessary, since if you have a valid token it means the user exists and has a valid session. I would recommend adding both the email and the user_id to the payload of the token and then use that